pkg/operator/ceph/controller/predicate.go

```diff
@@ -436,6 +436,29 @@ func WatchControllerPredicate() predicate.Funcs {
 				if isUpgrade {
 					return true
 				}
+
+			case *cephv1.CephCOSIDriver:
+				objNew := e.ObjectNew.(*cephv1.CephCOSIDriver)
+				logger.Debug("update event on CephCOSIDriver CR")
+				// If the labels "do_not_reconcile" is set on the object, let's not reconcile that request
+				IsDoNotReconcile := IsDoNotReconcile(objOld.GetLabels())
```

Comment:
Please follow trends from other predicates. This should be `objNew`, not `objOld`.

```suggestion
				IsDoNotReconcile := IsDoNotReconcile(objNew.GetLabels())
```